lib/pharos/phases/base.rb

```diff
@@ -1,17 +1,42 @@
 # frozen_string_literal: true
 
-require_relative 'logging'
+require 'logger'
 require_relative '../phases'
 
 module Pharos
   module Phases
     class Base
-      include Pharos::Phases::Logging
+      # @return [String]
+      def self.title
+        # XXX: prettier
+        self.name
```

Comment:
This currently returns the machine-readable `Pharos::Phases::ConfigureHost` name, needs to be humanized.